package.json

```diff
@@ -11,7 +11,37 @@
   "bin": {
     "setup-project": "./setup-script.js"
   },
+  "release": {
+    "branches": [
+      "main"
+    ],
+    "dryRun": true,
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      [
+        "@semantic-release/npm",
+        {
+          "npmPublish": false
```

Comment:
This is just a secondary flag to make sure we won't accidentally publish to npm while in test phase